Adyen/UI/List/ListItemView.swift

```diff
@@ -49,9 +49,14 @@ public final class ListItemView: UIView, AnyFormItemView {
                 subtitleLabel.adyen.apply(style.subtitle)
                 trailingTextLabel.adyen.apply(style.trailingText)
             }
+            if let isSelectable = item?.isSelectable, isSelectable {
+                self.addGestureRecognizer(itemGestureRecognizer)
```

Comment:
Do you think there is a way to use the `ListCell` instead of adding a gesture recognizer (thinking about the highlighting + accessibility that gets lost by using a gesture recognizer)?

Currently the `ListItem` is intended to only be used by the `ListViewController` and the full functionality might not be needed in this case.
Maybe even using a button that looks like a cell could be an option.